packs/node/pack.go

```diff
@@ -8,6 +8,11 @@ type Pack struct {
 	Analysis *Analysis
 }
 
+const (
+	nodeExpressStencilTemplatePath = "" // this way we only have to add the filename. We should start by download the templates.json, do a couples of checks and after that download the stuff
+	templateRepositoryBranch = "master"
```

Comment:
To enable testing, let this be a parameter into the CLI

Reply:
Do you mean add a -btr flag to pass it as a param to the starter command?